core/sql/ustat/hs_globals.cpp

```diff
@@ -4307,11 +4302,12 @@ Lng32 HSSample::make(NABoolean rowCountIsEstimate, // input
     // 2 cqds allow user control of these retries.
     Int32 centiSecs = getDefaultAsLong(USTAT_RETRY_DELAY);
     Int32 limit = getDefaultAsLong(USTAT_RETRY_LIMIT);
+    Int64 printPlan = 1;
     if (limit < 1 || centiSecs < 1) // user does not want any retry
```

Comment:
Can we use NABoolean here through out?  It is a little bit more efficient than Int64 when pass to a function.

Reply:
Yes, I wanted to do this. Unfortunately, there are over 100 calls to HSFuncExecQuery, so I chose not to do that now. (It also complicates the merge up into derivative products, as there are even more calls to HSFuncExecQuery there.)